history-of-the-quakers-v4/original/05-book-6-chapter-18.adoc

```diff
@@ -356,25 +386,35 @@ at which disappointment the judge was greatly incensed,
 and treated the grand jury with indecent asperity,
 addressing them in this passionate language:
 
-Gentlemen, surely you have forgot your oaths,
+[.small-break]
+'''
```

Comment:
how about instead of this [small-break] we make the next paragraph a [.discourse-part] as well, even though it doesn't start with the speaker.  It feels like it fits pretty well that way to me.  You can probably then also delete the quotes that you added, I think.